data-fusion/s1_flooding_visualisation/copernicus_browser.js

```diff
@@ -0,0 +1,36 @@
+//VERSION=3
+function setup() {
+    return {
+      input: [{
+          datasource: "S2L2A",
+          bands: ["B08", "B03", "B04"]
+        },
+        {
+          datasource: "S1GRD",
+          bands: ["VV", "VH", "dataMask"]
+        }
+      ],
+      output: { bands: 4 },
+      mosaicking: "SIMPLE"
+    };
+  }
+  
+  function toDB(input){
+    return 10 * Math.log(input)/Math.LN10;
```

Comment:
Let's put the spaces to make this consistent. For example, `10 * Math.log(input) / Math.LN10`. Please do check other lines to ensure the consistency.

Reply:
Done - so far I have directly taken the calculation code from the existing script by William, but the improvement is justified